honestcash-v2/src/app/modules/welcome/directives/tooltip.directive.ts

```diff
@@ -0,0 +1,88 @@
+import { Directive, Input, ElementRef, HostListener, Renderer2 } from '@angular/core';
```

Comment:
- Isn't there a library for that already?
- what about a license of the code?

We should possibly use libraries. Coping code (even jsfiddle) might be problematic because of the licensing.

- Isn't there a library for that already?
- what about a license of the code?

We should possibly use libraries. Coping code (even jsfiddle) might be problematic.

Maybe this? Or is there something better?
https://www.npmjs.com/package/ng-tippy